### ISSUE TITLE
Proton Manual, p. 10: A 6 MHz version of the Z80 is depicted, yet 8 MHz is stated as default clock speed.

### ISSUE THREAD

**Issue by JacekRuzyczka** [KALTCHUK/Z80_Modular_Computer#1]
The CPU visible on the photo of the PSU board is a Z84C0006PEC, which is rated for a clock speed of 6 MHz, not 8 MHz.

**Comment by KALTCHUK** [comment]
True. It's running overclocked. I tested it at 10MHz and it also worked
fine. At 12MHz it got crazy on the I/O requests. So, to keep it safe,I
chose to run at 8MHz.

Regards,

Eng.'. Ricardo Kaltchuk
+972 55 944 1669

**Comment by JacekRuzyczka** [comment]
OK, thx. You might know, though, that there are already 10 MHz and 20 MHz variants of the Z84C on the market.
